### ISSUE TITLE
Add axe scans to all e2e tests

### ISSUE THREAD

**Issue by LeKeve** [department-of-veterans-affairs/va.gov-team#25064]
## All End-to-End tests should include axe best practice checks

Some of our e2e tests describe blocks are missing axe best practice checks. We should add those checks to each `it()` block.

Discovered in #24021

## Proposed Solution (if known)
Add axe checks to each e2e test block

## Acceptance Criteria
- [ ] Axe best practice checks are included in each e2e test block

## Definition of Done
- [ ] Fix and/or document decisions made.
